sky/sdk/lib/widgets/block_viewport.dart

```diff
@@ -86,16 +134,8 @@ class BlockViewport extends RenderObjectWrapper {
     super.didUnmount();
   }
 
-  // _offsets contains the offsets of each child from the top of the
-  // list up to the last one we've ever created, and the offset of the
-  // end of the last one. If there's no children, then the only offset
-  // is 0.0.
-  List<double> _offsets = <double>[0.0];
-  int _currentStartIndex = 0;
-  int _currentChildCount = 0;
-  bool _didReachLastChild = false;
-
   int _findIndexForOffsetBeforeOrAt(double offset) {
+    final List<double> _offsets = layoutState._childOffsets;
```

Comment:
It's probably better for this local variable to have a normal name (i.e., without the _) even though that means editing more code.